portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsColorField.testcase

```diff
@@ -153,14 +153,46 @@ definition {
 	}
 
 	@description = "Verify that a Predefined Value can be set for the Color Field"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test PredefinedValueCanBeAdded {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133144 PredefinedValueCanBeAdded pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Color");
+
+		Form.gotoAdvancedTab();
+
+		FormViewBuilder.setFieldPredefinedDate(fieldValue = "7F26FF");
```

Comment:
Hi @fmarcio , even though this macro does what the test needs. Could you please replicate the behavior of this macro to another one with a proper name, because this one it is supposed to set a Date. You can create this macro in `FormFields.macro`. Thanks in advance.

Reply:
Actually, there is already a macro called `editPredefinedValue` in the `FormFields.macro`. Maybe you could use it.

Reply:
> Actually, there is already a macro called `editPredefinedValue` in the `FormFields.macro`. Maybe you could use it.

@mlan0 requests done! But the macro `editPredefinedValue` did not work on my test. So I created a new macro and path that replicates the behavior of the first one I used.